Modules/Field/angle.py

```diff
@@ -0,0 +1,15 @@
+from . import config
+'''
+Input - the time passed from measurement start
+Output - the ratio of the route that the satellite has already passed
```

Comment:
"The ratio of the route" ?
Can you rephrase that a bit ?

Reply:
I believe he meant the quotient of the length the satellite has already traveled to the entire trajectory length.

Reply:
You can also see that as "The proportion of the satellite orbit that was completed"

Reply:
To [ complete the sentence ]

Proportion has to consist of two measures

Reply:
So percentage,
or the proprtion of completed orbit out of the entire orbit.

Reply:
:+1: